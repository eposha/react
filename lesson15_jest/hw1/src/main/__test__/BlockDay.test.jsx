import React from "react";
import { shallow } from "enzyme";
import BlockDay from "../BlockDay";

describe("<BlockDay/>", () => {
  it("should display BlockDay", () => {
    const wrappedComponent = shallow(<BlockDay />);
    expect(wrappedComponent).toMatchSnapshot();
  });
});
