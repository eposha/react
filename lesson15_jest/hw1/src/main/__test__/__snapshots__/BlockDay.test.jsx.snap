// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlockDay/> should display BlockDay 1`] = `
Array [
  <div
    className="block-day"
    key="0"
  >
    <BlockHours
      hourId="2020-02-17"
    />
  </div>,
  <div
    className="block-day"
    key="1"
  >
    <BlockHours
      hourId="2020-02-17"
    />
  </div>,
  <div
    className="block-day"
    key="2"
  >
    <BlockHours
      hourId="2020-02-17"
    />
  </div>,
  <div
    className="block-day"
    key="3"
  >
    <BlockHours
      hourId="2020-02-17"
    />
  </div>,
  <div
    className="block-day"
    key="4"
  >
    <BlockHours
      hourId="2020-02-17"
    />
  </div>,
  <div
    className="block-day"
    key="5"
  >
    <BlockHours
      hourId="2020-02-17"
    />
  </div>,
  <div
    className="block-day"
    key="6"
  >
    <BlockHours
      hourId="2020-02-17"
    />
  </div>,
]
`;
